.gitignore

```diff
@@ -57,7 +57,9 @@ target/
 # PyCharm
 .idea
 
+test_output
 rhOverM_Asymptotic*h5
+rh_FiniteRadii_CodeUnits.h5
```

Comment:
These changes can be reverted if you use my suggestion of temporary directories.